README.md

```diff
@@ -1,54 +1,46 @@
-[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/nr-frontend-starting-app)
+# Getting Started with Create React App
```

Comment:
I think you can keep the previous readme version and improve it with new commands if needed, maybe consider adding them to the contributing guide instead.

Reply:
Agree, but I don't think it is necessary to add on the contributing guide, those are just some generic information about CRA. I adjusted the readme and left the contributing as it is, hope that's ok :p